### PR TITLE
Wallet func

### PR DESCRIPTION
**Pull request by yuanshanhshan**
rm transaction public service

### CODE REVIEW

File: api/verifier/v1/transaction.proto [hunk 1]
```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
```
Comment (xiaoxiaff) [inline api/verifier/v1/transaction.proto:1]: still need this?

File: docs/TODO.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# TODO
```
Comment (xiaoxiaff) [inline docs/TODO.md:1]: db can come as the lowest priority. Most of state should be persistent on-chain until we have p2p connections for verifiers.

File: internal/biz/transaction.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package biz
+
+import (
+	"context"
+	"github.com/go-kratos/kratos/v2/log"
+	"time"
+)
```
Comment (xiaoxiaff) [inline internal/biz/transaction.go:7]: small nit: grouping as native pkgs + 3rd party pkgs + internal pkgs.

```suggestion
import (
	"context"
	"time"

	"github.com/go-kratos/kratos/v2/log"
)
```

File: internal/conf/conf.proto [hunk 4]
```diff
@@ -66,4 +66,7 @@ message Contract {
 
 message Wallet {
   string private_key = 1;
+  string key = 2;
+  string iv = 3;
```
Comment (xiaoxiaff) [inline internal/conf/conf.proto:70]: iv stands for?

File: internal/data/transaction.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package data
+
+import (
+	"context"
+	"github.com/carv-protocol/verifier/internal/biz"
+	"github.com/go-kratos/kratos/v2/log"
```
Comment (xiaoxiaff) [inline internal/data/transaction.go:6]: ditto

File: internal/worker/chain.go [hunk 6]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"encoding/hex"
+	"fmt"
+	"github.com/carv-protocol/verifier/pkg/sm4"
 	"math/big"
 	"strings"
 	"sync"
```
Comment (xiaoxiaff) [inline internal/worker/chain.go:11]: ditto

File: pkg/sm4/m2.go [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+//	fmt.Printf("SM4 encode result: %x\n", ciphertxt)
+//	fmt.Println("SM4 encode result :\n", base64.StdEncoding.EncodeToString(ciphertxt))
+//
+//	res, err := sm4Decrypt(key, iv, ciphertxt)
+//	if err != nil {
+//		log.Fatal(err)
+//	}
+//	fmt.Println("SM4 decode text ：", string(res))
+//
+//}
```
Comment (xiaoxiaff) [inline pkg/sm4/m2.go:67]: Remove this. If we need a place for testing then we can put into a separate cmd.

File: api/verifier/v1/transaction.proto (hunk 1 above)
Comment (yuanshanhshan) [reply api/verifier/v1/transaction.proto:1]: Removed

File: docs/TODO.md (hunk 2 above)
Comment (yuanshanhshan) [reply docs/TODO.md:1]: Get it

File: internal/biz/transaction.go (hunk 3 above)
Comment (yuanshanhshan) [reply internal/biz/transaction.go:7]: Get it

File: internal/data/transaction.go (hunk 5 above)
Comment (yuanshanhshan) [reply internal/data/transaction.go:6]: Get it

File: pkg/sm4/m2.go (hunk 7 above)
Comment (yuanshanhshan) [reply pkg/sm4/m2.go:67]: Removed

**Pull request closed without merging**